MediaBrowser.Api/Sessions/ApiKeyService.cs

```diff
@@ -0,0 +1,85 @@
+using System;
+using System.Globalization;
+using MediaBrowser.Controller;
+using MediaBrowser.Controller.Configuration;
+using MediaBrowser.Controller.Net;
+using MediaBrowser.Controller.Security;
+using MediaBrowser.Controller.Session;
+using MediaBrowser.Model.Services;
+using Microsoft.Extensions.Logging;
+
+namespace MediaBrowser.Api.Sessions
+{
+    [Route("/Auth/Keys", "GET")]
+    [Authenticated(Roles = "Admin")]
+    public class GetKeys
```

Comment:
While you are at it, maybe move these classes into their own files?

Reply:
That just makes it more annoying imo

Reply:
I agree, these work fine in a single file.

Reply:
Ok, in that case you should disable the analyzer warnings for multiple classes in 1 file for this file

Reply:
We should probably disable it for both API projects in that case.

Reply:
Not the new one

Reply:
@Bond-009 can I leave that for a warning pull request?